nsis/gvim.nsi

```diff
@@ -139,7 +139,7 @@ RequestExecutionLevel highest
 # Show all languages, despite user's codepage:
 !define MUI_LANGDLL_ALLLANGUAGES
 # Always show dialog choice language
-#!define MUI_LANGDLL_ALWAYSSHOW
+!define MUI_LANGDLL_ALWAYSSHOW
```

Comment:
Why this is changed?

Reply:
Thanks for noticing.
I turned it on for testing. i'll have to clean it up now.